EE guides: add run_community_ee_image.rst (#222)
ansible/ansible-documentation: docs/docsite/rst/getting_started_ee/run_execution_environment.rst

  There are other tools besides ``ansible-navigator`` you can run EEs with.

.. note::

  To try an EE without building a custom one, substitute ``postgresql_ee`` image used in this document with the ``ghcr.io/ansible/community-ee:latest`` community image.

> 

Reviewer: Is this a necessary note when you have a separate page already for running with just the community ee?

Author: i think you're right, it's not necessary

>   If you only need to try out EEs without building your custom one, you can simply substitute ``postgresql_ee`` image used in this document with the ``ghcr.io/ansible/community-ee:latest`` community image.

Reviewer 2: ```suggestion
  To try an EE without building a custom one, substitute ``postgresql_ee`` image used in this document with the ``ghcr.io/ansible/community-ee:latest`` community image.
```

> What to read next

Reviewer: Change this to a seealso as well please

Author: Done

-----------------

* More about the `EE definition file <https://ansible-builder.readthedocs.io/en/stable/definition/>`_ and available options
* `Ansible Builder CLI usage <https://ansible-builder.readthedocs.io/en/stable/usage/>`_

> * `Ansible Navigator official documentation <https://ansible-navigator.readthedocs.io/>`_

Reviewer: ```suggestion
* `Ansible Navigator documentation <https://ansible-navigator.readthedocs.io/>`_
```
